docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,161 @@ weight = -17
+
+As a replacement for PhantomJS we’ve developed the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) which is a plugin that runs on the backend and handles rendering panels and dashboards as PNG images using headless Chromium/Chrome. The [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can either be installed as a Grafana plugin running in its own process side-by-side with Grafana. or run as an external HTTP service, hosted using Docker or as a standalone application.
+
+Read more about [Image Rendering]({{< relref "../administration/image_rendering/" >}}) in the documentation for further instructions.
+
+## Rich History in Explore out of beta
+
+## Stackdriver data source supports Service Monitoring
+
+[Service monitoring](https://cloud.google.com/service-monitoring) in Google Cloud Platform (GCP) enables you to monitor based on Service Level Objectives (SLOs) for your GCP services. The new SLO query builder in the Stackdriver data source allows you to display SLO data in Grafana.
+
+https://grafana.com/docs/grafana/latest/features/datasources/stackdriver/#slo-service-level-objective-queries
+
+## Time zone support
+
+### Grafana Enterprise
+
+Grafana Enterprise focuses on solving problems for large companies and Grafana installations. And in Grafana 7.0 we are finally
+solving one of the most common problems of using Grafana at scale.
+
+That is problems like:
+
+- There are too many dashboards, how do I find the right one?
+- How to find popular dashboards
+- How to find dashboards with errors
+- How to identify dashboards that are not being used
+- Who created or last viewed this dashboard?
+
+## Usage Analytics and Presence
+
+In Grafana 7.0 we are introducing a series of features that build on our new usage analytics engine. Features like improved dashboard search to sort dashboards by usage and errors. When a user opens a dashboard, they will see a presence indicator of who else is viewing the same dashboard. And finally open a dashboard usage stats drawer to view recent dashboard usage.
+
+{{< docs-imagebox img="/img/docs/v70/presence_indicator.jpg" max-width="1024px" caption="Grafana Enterprise - Presence Indicator" >}}
+{{< docs-imagebox img="/img/docs/v70/dashboard_insights_stats.png" max-width="1024px" caption="Dashboard Insights Stats" >}}
+{{< docs-imagebox img="/img/docs/v70/dashboard_insights_users.png" max-width="1024px" caption="Dashboard Insights Users" >}}
+{{< docs-imagebox img="/img/docs/v70/improved_search.png" max-width="1024px" caption="Search" >}}
+
+## Upgrading
+
+See [upgrade notes]({{< relref "../installation/upgrading/#upgrading-to-v7-0" >}}).
+
+## Changelog
```

Comment:
```suggestion
## Change log
```

Reply:
I don't agree with this: https://en.wikipedia.org/wiki/Changelog `changelog` is the established term.

Reply:
Okeydokey